content/organization/events/speakers/index.en.md

```diff
@@ -27,6 +27,47 @@ weight: 53
 
 * Remember the [rules about Prioritization of Underrepresented/Minority Genders](/about/mission/#r-ladies-rules--guidelines).
 
+* In line with R-Ladies’ Mission Statement and with our [Black Lives Matter Statement](https://blog.rladies.org/post/2020-06-06-blm/) 
+to achieve participation and representation in our events of generally excluded groups, taking into account the intersectionality of gender, race, language and geography; for these reasons we request:
+
+  - In chapter events, invite speakers of diverse race, language and geography (i.e., avoid a one year period with all white speakers at the chapter events).
+  - For panel discussions of 3 or more panelists, invite speakers of diverse race, language and geography (i.e., avoid all white panels).
+
+### How to find speakers? 
+- You can use the [R-Ladies Directory](https://rladies.org/directory/) to identify potential speakers that are not: white, native anglophone, or from high income level countries.
+- You can reach other communities to request for speakers information (see lists below).
+- You can reach other R-Ladies in the R-Ladies Community Slack.
+- You can use others directories to look for speakers (see list below). 
+- For online events, we encourage local chapters to reach out beyond their local community to invite R-Ladies from different parts of the world as speakers.
+### Other communities to reach
+
+* [Minorities in R](https://mircommunity.com/)
+
+* [Black Girls Code](https://www.blackgirlscode.com/)
+
+* [Black in Data](https://www.blackindata.co.uk/)
+
+* [Black in Data Week](https://blkindata.github.io/)
+
+* [Coding Black Female](https://codingblackfemales.com/)
+
+### Other womens speakers directories
+
+* [500 Scientific Women](https://500womenscientists.org/request-a-scientist)
+
+* [Diversifica](https://diversifyeeb.com)
+
+* [Women in geospatial](https://www.womeningeospatial.org/join/speakers-database)
+
+* [Find your speakers](https://speakerinnen.org/)
+
+* [Survey of scientists from Women in Bioinformatics and Data Science LatAm](https://t.co/nI8rHED8ry?amp=1) 
+
+* * [Directory of technology speakers](https://github.com/lasdesistemas/speakers-tecnologia) from [Las de Sistemas - LatAm](https://twitter.com/lasdesistemas/status/1273374539888672774?s=20): 
+ 
+
+We need to do for this minorities group inside our community, the same we request other communities and spaces do for womens.  
```

Comment:
```suggestion
We need to do for the minority groups inside our community the same that we request other communities and spaces do for women and gender minorities.  
```

Reply:
I would move this sentence up, probably to line 35 to end that last bullet point with the two sub-bullets. It's an important sentence and I think it's easily overlooked in its current place.